### DISCUSSION TITLE
DOOM captcha instead of hCaptcha

### DISCUSSION THREAD

**Discussion by privacytoolsIO** [stoatchat/discussions#346] [Feature Suggestions]
Have u guys considered including the DOOM captcha instead of the hCaptcha? Since the target group are gamers and i'm kinda sick of selecting sea planes daily. it's open source: https://vivirenremoto.github.io/doomcaptcha/

**Comment by Rexogamer** [comment]
The target group for Revolt isn't gamers though - and, whilst a bit of fun, the site for that does explicitly state that it can be "easily broken"; a well built and preferably privacy focused captcha is what we need to prevent spam and such
